mypy/erasetype.py

```diff
@@ -87,7 +87,9 @@ def visit_literal_type(self, t: LiteralType) -> ProperType:
 
     def visit_union_type(self, t: UnionType) -> ProperType:
         erased_items = [erase_type(item) for item in t.items]
-        return UnionType.make_simplified_union(erased_items)
+        from mypy.typeops import make_simplified_union  # asdf
+        # XXX: does this need to be simplified?
```

Comment:
I think sometimes it may be useful. For example we probably want `Union[List[T], List[S]]` become just `List[Any]`, not `Union[List[Any], List[Any]]`. I think a similar applies to the question below.